.github/workflows/ci-legacy.yml

```diff
@@ -0,0 +1,30 @@
+name: LEGACY
+
+on: push
+
+jobs:
+  test:
+    name: Test on Node ${{ matrix.node }} and ${{ matrix.os }}
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        node: ["0.8"]
```

Comment:
Is it not possible to test `v0.6`?

To be fully bulletproof on this level, we should test all major since v0.6 (so also v0.10, v0.12, v4, v6 and v8)

Reply:
I tried doing multiple builds yesterday on github actions also on DO server but sadly all I got was segment fault errors same is happening on Travis CI as well if you see current build logs

Here's commit history of old repo https://github.com/tiny-leaf/querystring-old/commits/node-0.6-test

Reply:
Ok, thanks for investigating that. If that's not possible, we need to stick to 0.8 as you've configured

Reply:
Still, can we have v0.10, v0.12, v4, v6 and v8 covered?

Reply:
Done